kafka-client/src/main/java/dev/responsive/kafka/api/stores/ResponsiveStores.java

```diff
@@ -62,7 +64,7 @@ public static KeyValueBytesStoreSupplier keyValueStore(final ResponsiveKeyValueP
    *
    * @param name the store name
    * @return a supplier for a key-value store with the given options
-   *         that uses Responsive's storage for its backend
+   * that uses Responsive's storage for its backend
```

Comment:
Is your IDE set to autoformat? I usually turn that off just because it can make an already-long PR even longer -- but it's not a big deal. To me, at least -- if you submit a PR to Kafka/Kafka Streams, some of the other committers _will_ complain about extraneous formatting changes. Even if they are an improvement lol. Just a heads up